pkgs/applications/editors/vscode/extensions/default.nix

```
@@ -2047,6 +2047,8 @@ let
         };
       };
 
+      QURI.vscode-squiggle = callPackage ./squiggle { };
```

```suggestion
      QURI.vscode-squiggle = callPackage ./squiggle { };
```
Upstream uses name as: `QURI.vscode-squiggle`
Ideally, we should follow up with that.

---

```suggestion
      quri.vscode-squiggle = callPackage ./quri.vscode-squiggle { };
```
we lowercase things